wpsc-admin/includes/display-items-functions.php

```diff
@@ -127,12 +127,10 @@ function wpsc_price_control_forms() {
 	}
 
 	$product_data['meta']['_wpsc_price'] = wpsc_format_number( $product_data['meta']['_wpsc_price'] );
-
-	$currency_data = $wpdb->get_results( "SELECT * FROM `" . WPSC_TABLE_CURRENCY_LIST . "` ORDER BY `country` ASC", ARRAY_A );
+	$currency_data = wpsc_get_all_countries();
 
 	/* Get country name and symbol */
-	$currency_type = get_option( 'currency_type' );
-	$country       = new WPSC_Country( $currency_type );
+	$country       = wpsc_get_currency_type_country_object();
```

Comment:
I'm not a massive fan of ending function names in data types, e.g. `WPSC_Countries::get_countries_array()` or this, `wpsc_get_currency_type_country_object()`. Seems to me it'd be better to just be `wpsc_get_currency_type_country`?